components/ucd/category/tests/unicode_version_tests.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2017 The UNIC Project Developers.
+//
+// See the COPYRIGHT file at the top-level directory of this distribution.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+
+extern crate unic_ucd_core;
+extern crate unic_ucd_category;
+
+
+//#[test]
```

Comment:
Whoops, need to re-enable this (had it disabled before the python build script was edited).